### ISSUE TITLE
print_info() function

### ISSUE THREAD

**Issue by rakovskaiaia** [3fon3fonov/exostriker#131]
Hello! I am using exostriker with jupyter notebook. Which function can I use instead of print_info() which is not working right now?

**Comment by 3fon3fonov** [comment]
Yes, print_info() was removed. Are you using some of the examples .py files in the "examples" directory? These need to be updated into the new API. Sorry for not being able to do it at this point. 

What exactly do you want to print, so I will give you some directions?

**Comment by rakovskaiaia** [comment]
Yes, I saw that function in examples. I need to print planetary parameters of orbit (P,K,e,w,Ma) with uncertainties
